libs/components/forms/src/lib/modules/radio/radio-group.component.spec.ts

```diff
@@ -559,6 +559,33 @@ describe('Radio group component (reactive)', function () {
     await fixture.whenStable();
     await expectAsync(fixture.nativeElement).toBeAccessible();
   });
+
+  it('should set aria-owns as a space-separated list of radio ids', () => {
+    fixture.detectChanges();
+
+    const radioGroupEl: HTMLDivElement =
+      fixture.nativeElement.querySelector('.sky-radio-group');
+
+    expect(radioGroupEl.getAttribute('aria-owns')).toMatch(
+      /sky-radio-sky-radio-[0-9]+-input sky-radio-sky-radio-[0-9]+-input sky-radio-sky-radio-[0-9]+-input/
+    );
+  });
+
+  it('should update aria-owns if radio IDs change', () => {
```

Comment:
I think this test wording and the test could be improved. How are radio IDs changing? Adding new radios, removing old radios, both? And how exactly should the value change? 

If this were to fail 6 months from now it would take me a while to figure out what was supposed to be happening.

Reply:
Updated the wording.